snuba/admin/static/clickhouse_migrations/index.tsx

```diff
@@ -17,6 +17,15 @@ function ClickhouseMigrations(props: { api: Client }) {
   const [migrationId, setMigrationId] = useState<string | null>(null);
   const [SQLText, setSQLText] = useState<string | null>(null);
 
+  const [header, setHeader] = useState<string | null>(null);
+  const [show_action, setShowAction] = useState<boolean | null>(false);
+
+  const dry_run_header = "Raw SQL for running a migration (forwards) or reversing \n (backwards). Good to do before executing a migration for real."
```

Comment:
I think this text should always be present whenever the buttons are present, so I would move this text to display above the buttons. You could change it a little to be like:
`Before executing a migration, do a dry run first. This will generate the raw SQL for running a migration (forwards) or reversing (backwards) a migration so that you can verify it's contents.`

Something that basically tells them why they are doing this first.

But I do think it makes sense to have a header over the output textbox to indicate whether it's the raw sql from the dry run or if its the output from actually executing. so maybe you could just have that header be `Dry run SQL output`

Reply:
makes sense. updated